### PR TITLE
Add mysql database to docker-compose

### PR DESCRIPTION
**Pull request by donce**

### CODE REVIEW

**Comment by ignasbernotas** [comment]
LGTM

File: docker-compose.yml [hunk 1]
```diff
@@ -4,3 +4,10 @@ services:
     build: .
```
Comment (Waldz) [inline docker-compose.yml]: Inject environment variables, which points to database container name

File: README.md [hunk 2]
```diff
@@ -2,36 +2,39 @@
 
 API for Node & Client
 
+## Setup [![pyVersion27](https://img.shields.io/badge/python-2.7-blue.svg)](https://www.python.org/download/releases/2.7/)
 
+- Setup `settings.py` file:
```
Comment (Waldz) [inline README.md]: If all variables are read from environment, usually configuring them in files like `settings.py` is not need anymore

Comment (Waldz) [reply README.md]: Starting development environment should be be easy as:
```
git clone ...
docker-compose up
```

**Pull request merged**